internal/templates/signald_deployment.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.Name }}
```

I think the name template could just be here

```suggestion
  name: {{ .Values.Name }}-signald
```